gensim/similarities/docsim.py

```diff
@@ -1015,7 +1015,7 @@ class WmdSimilarity(interfaces.SimilarityABC):
         >>>
         >>> model = Word2Vec(common_texts, vector_size=20, min_count=1)  # train word-vectors
         >>>
-        >>> index = WmdSimilarity(common_texts, model)
+        >>> index = WmdSimilarity(common_texts, model.mv)
```

Comment:
Ping @DingQK

Reply:
Hi @piskvorky , 

Please forgive me for my late response. I believe this is not a typo because the second input is class KeyedVectors (As i quote my earlier reply). 

> > What documentation? And what inconsistency? I don't follow.
> 
> @piskvorky Sorry for my unclear description. The documentation I am referring to is https://radimrehurek.com/gensim/similarities/docsim.html#gensim.similarities.docsim.WmdSimilarity. The documentation's directory in this repo is `docs/similarities/docsim.html`.
> 
> In the documentation, the second input of the `init` function of `WmdSimilarity` is `Word2Vec()`. But in `gensim/test/test_similarities.py`, the unit tests (`TestWmdSimilarity`) for `WmdSimilarity` uses `Word2Vec().wv` as the second input. I am guessing through the development of WmdSimilarity, the version of `WmdSimilarity.__init__` and `TestWmdSimilarity` are not updated simultaneously.
> 
> If this is an inconsistency, which usage should I follow and may I fix this issue? Thank you so much for your time!

Reply:
Hi @piskvorky ,

So so sorry for my delay in response. I quote two of my earlier response as follows. I believe this is not a typo maybe?


> > What documentation? And what inconsistency? I don't follow.
> 
> @piskvorky Sorry for my unclear description. The documentation I am referring to is https://radimrehurek.com/gensim/similarities/docsim.html#gensim.similarities.docsim.WmdSimilarity. The documentation's directory in this repo is `docs/similarities/docsim.html`.
> 
> In the documentation, the second input of the `init` function of `WmdSimilarity` is `Word2Vec()`. But in `gensim/test/test_similarities.py`, the unit tests (`TestWmdSimilarity`) for `WmdSimilarity` uses `Word2Vec().wv` as the second input. I am guessing through the development of WmdSimilarity, the version of `WmdSimilarity.__init__` and `TestWmdSimilarity` are not updated simultaneously.
> 
> If this is an inconsistency, which usage should I follow and may I fix this issue? Thank you so much for your time!



> Hi @piskvorky , I check the code closely and I believe the documentation needs an update as the second input is class KeyedVectors. I updated the documentation and the annotation code. Also, I add a unit test for this str method in test_similarities.py. Thank you so much!

Reply:
Even so, `mv` != `wv`.

Reply:
OMG! That is such a huge mistake of me. Sorry for that!

Reply:
Can you run the example in Python, end-to-end? To make sure it really works.

Reply:
I run the example as requested, and the result is as followed:
![image](https://user-images.githubusercontent.com/58072531/157635160-d68ee9b8-b2c8-4232-8380-b6ac03650d43.png)
I am not sure if this meets the requirement~